.github/buildx-releases.json

```diff
@@ -1,106 +1,43 @@
-      "https://github.com/docker/buildx/releases/download/v0.19.0/buildx-v0.19.0.linux-arm-v6.sbom.json",
-      "https://github.com/docker/buildx/releases/download/v0.19.0/buildx-v0.19.0.linux-arm-v7",
-      "https://github.com/docker/buildx/releases/download/v0.19.0/buildx-v0.19.0.linux-arm-v7.provenance.json",
-      "https://github.com/docker/buildx/releases/download/v0.19.0/buildx-v0.19.0.linux-arm-v7.sbom.json",
-      "https://github.com/docker/buildx/releases/download/v0.19.0/buildx-v0.19.0.linux-arm64",
-      "https://github.com/docker/buildx/releases/download/v0.19.0/buildx-v0.19.0.linux-arm64.provenance.json",
-      "https://github.com/docker/buildx/releases/download/v0.19.0/buildx-v0.19.0.linux-arm64.sbom.json",
-      "https://github.com/docker/buildx/releases/download/v0.19.0/buildx-v0.19.0.linux-ppc64le",
-      "https://github.com/docker/buildx/releases/download/v0.19.0/buildx-v0.19.0.linux-ppc64le.provenance.json",
-      "https://github.com/docker/buildx/releases/download/v0.19.0/buildx-v0.19.0.linux-ppc64le.sbom.json",
-      "https://github.com/docker/buildx/releases/download/v0.19.0/buildx-v0.19.0.linux-riscv64",
-      "https://github.com/docker/buildx/releases/download/v0.19.0/buildx-v0.19.0.linux-riscv64.provenance.json",
-      "https://github.com/docker/buildx/releases/download/v0.19.0/buildx-v0.19.0.linux-riscv64.sbom.json",
-      "https://github.com/docker/buildx/releases/download/v0.19.0/buildx-v0.19.0.linux-s390x",
-      "https://github.com/docker/buildx/releases/download/v0.19.0/buildx-v0.19.0.linux-s390x.provenance.json",
-      "https://github.com/docker/buildx/releases/download/v0.19.0/buildx-v0.19.0.linux-s390x.sbom.json",
-      "https://github.com/docker/buildx/releases/download/v0.19.0/buildx-v0.19.0.openbsd-amd64",
-      "https://github.com/docker/buildx/releases/download/v0.19.0/buildx-v0.19.0.openbsd-amd64.provenance.json",
-      "https://github.com/docker/buildx/releases/download/v0.19.0/buildx-v0.19.0.openbsd-amd64.sbom.json",
-      "https://github.com/docker/buildx/releases/download/v0.19.0/buildx-v0.19.0.openbsd-arm64",
-      "https://github.com/docker/buildx/releases/download/v0.19.0/buildx-v0.19.0.openbsd-arm64.provenance.json",
-      "https://github.com/docker/buildx/releases/download/v0.19.0/buildx-v0.19.0.openbsd-arm64.sbom.json",
-      "https://github.com/docker/buildx/releases/download/v0.19.0/buildx-v0.19.0.windows-amd64.exe",
-      "https://github.com/docker/buildx/releases/download/v0.19.0/buildx-v0.19.0.windows-amd64.provenance.json",
-      "https://github.com/docker/buildx/releases/download/v0.19.0/buildx-v0.19.0.windows-arm64.exe",
-      "https://github.com/docker/buildx/releases/download/v0.19.0/buildx-v0.19.0.windows-arm64.provenance.json",
-      "https://github.com/docker/buildx/releases/download/v0.19.0/buildx-v0.19.0.windows-arm64.sbom.json",
-      "https://github.com/docker/buildx/releases/download/v0.19.0/checksums.txt"
-    ]
-  },
-  "v0.19.0": {
```

Comment:
do we still want to keep `v0.19.0` and only revert the `latest` to point to v0.18.0 ?

Reply:
With current GHA updating this releases json we can't but will make some changes to handle that.